docs/methodology.md

```diff
@@ -70,7 +70,8 @@ We test two possible runtime scenarios for each tool:
 ## Metrics collected
 ### Ingest time
 This measures the time taken to ingest the data. Smaller values are better indicating faster
-ingestion performance.
+ingestion performance. The ingest speed is calculated by dividing the on-disk size of the 
+pre-ingestion data by the ingestion time.
```

Comment:
Unrelated change?

Reply:
Yea, unrelated but minor. Maybe just get rid of it.

Reply:
Yeah, as our contribution guidelines say:

> Do limit the blast radius of your commit 

Generally, unrelated changes should be at least a little bit related (e.g., in the same file, etc.) These two are totally unrelated, lol.

Feel free to submit another PR for editing the docs though.

Reply:
reverted in next commit